components/datatypes/consentpreferences.schema.json

```diff
@@ -0,0 +1,337 @@
+{
+  "meta:license":[
+     "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+     "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+     "you may not use this file except in compliance with the License. You may obtain a copy",
+     "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":"https://ns.adobe.com/xdm/datatypes/consent-preferences",
+  "$schema":"http://json-schema.org/draft-06/schema#",
+  "title":"Privacy/Marketing Preferences (Consent)",
+  "description":"This schema captures privacy, personalization and marketing preferences (consents).",
+  "type":"object",
+  "meta:extensible":true,
+  "meta:abstract":true,
+  "definitions":{
+    "consentValue": {
+      "title": "Consent Value",
+      "description": "The value of the specific privacy/consent preference.",
+      "type": "string",
+      "enum": [
+        "yes",
+        "no",
+        "pending",
+        "unknown",
+        "not_applicable"
+      ],
+      "meta:enum": {
+        "yes": "The user has explicitly opted in to this feature",
+        "no": "The user has explicitly opted out of this feature",
+        "pending": "Pending verification",
+        "unknown":"Unknown",
+        "not_applicable":"Not Applicable"
+      }
+    },
+    "basisOfProcessing": {
+      "title": "Basis Of Processing",
+      "type": "string",
+      "description": "Basis of Processing",
+      "enum": [
+        "consent",
+        "legitimate_interest",
+        "contract",
+        "compliance",
+        "vital_interest",
+        "public_interest"
+      ],
+      "meta:enum": {
+        "consent": "User Consent",
+        "legitimate_interest": "Legitimate Interest",
+        "contract": "Contract",
+        "compliance": "Compliance with a Legal Obligation",
+        "vital_interest": "Vital Interest of the Individual",
+        "public_interest": "Public Interest"
+      }
+    },
+    "timestamp": {
+      "title": "Preference timestamp",
+      "description": "Timestamp of this specific opt out or preference.",
+      "type": "string",
+      "format": "date-time"
+    },
+    "consentOption": {
+      "type": "object",
+      "properties": {
+        "xdm:choice": {
+          "$ref": "#/definitions/consentValue"
+        },
+        "xdm:basisOfProcessing": {
+          "$ref": "#/definitions/basisOfProcessing"
+        },
+        "xdm:timestamp": {
+          "$ref": "#/definitions/timestamp"
```

Comment:
use `dc:date` instead

Reply:
@lrosenthol  where is dc:date defined? How do I use it?

Reply:
`dc:date` is one of the standard XMP values - it means a generic date and/or time.

Reply:
@lrosenthol  - Can you provide a link here to the standard definition of all the dc:* and repo:* fields that are used in XDM?